### PR TITLE
tepl 5.0.0, gnome-latex 3.38.0 and gedit 3.38.0

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/tepl.rb [hunk 1]
```diff
@@ -71,7 +79,7 @@ def install
       -I#{gtksourceview4.opt_include}/gtksourceview-4
       -I#{gtkx3.opt_include}/gtk-3.0
       -I#{harfbuzz.opt_include}/harfbuzz
-      -I#{include}/tepl-4
+      -I#{include}/tepl-5
```
Comment (SMillerDev) [inline Formula/tepl.rb:82]: This should work right @SeekingMeaning ?
```suggestion
      -I#{include}/tepl-#{version.major}
```

Comment (SeekingMeaning) [reply Formula/tepl.rb:82]: Yes

File: Formula/gedit.rb [hunk 2]
```diff
@@ -97,7 +96,7 @@ def post_install
       -I#{gtksourceview4.opt_include}/gtksourceview-4
       -I#{gtkx3.opt_include}/gtk-3.0
       -I#{harfbuzz.opt_include}/harfbuzz
-      -I#{include}/gedit-3.36
+      -I#{include}/gedit-3.38
```
Comment (SeekingMeaning) [inline Formula/gedit.rb:99]:
```suggestion
      -I#{include}/gedit-#{version.major_minor}
```

Comment (tschoonj) [reply Formula/gedit.rb:99]: Bad idea, Gedit actually rarely bumps this number. It's quite exceptional that it has been bumped twice in a row.

File: Formula/tepl.rb [hunk 3]
```diff
@@ -98,7 +106,7 @@ def install
       -lgio-2.0
       -lglib-2.0
       -lgobject-2.0
-      -ltepl-4
+      -ltepl-5
```
Comment (SeekingMeaning) [inline Formula/tepl.rb:109]:
```suggestion
      -ltepl-#{version.major}
```

File: Formula/tepl.rb [hunk 4]
```diff
@@ -17,16 +16,25 @@ class Tepl < Formula
...
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "amtk"
   depends_on "gtksourceview4"
   depends_on "uchardet"
 
+  # Submitted upstream at https://gitlab.gnome.org/GNOME/tepl/-/merge_requests/8
+  patch do
+    url "https://gitlab.gnome.org/GNOME/tepl/-/commit/a8075b0685764d1243762e569fc636fa4673d244.patch"
+    sha256 "cf4966f9975026ad349eac05980bdbc6cdfc2ed581b04c099ed892777db0767c"
```
Comment (SeekingMeaning) [inline Formula/tepl.rb:29]: (For full index)

```suggestion
    url "https://gitlab.gnome.org/GNOME/tepl/-/commit/a8075b0685764d1243762e569fc636fa4673d244.diff"
    sha256 "b5d646c194955b0c14bbb7604c96e237a82632dc548f66f2d0163595ef18ee88"
```

Comment (SeekingMeaning) [reply Formula/tepl.rb:29]: Actually nevermind, see Homebrew/brew#8701

**Comment by tschoonj** [comment]
Merge this one in?

File: Formula/gedit.rb (hunk 2 above)
Comment (SMillerDev) [inline Formula/gedit.rb:99]: Can we do `#{version.major}.#{version.minor}` here

File: Formula/gedit.rb [hunk 5]
```diff
@@ -122,7 +121,7 @@ def post_install
       -lcairo-gobject
       -lgdk-3
       -lgdk_pixbuf-2.0
-      -lgedit-3.36
+      -lgedit-3.38
```
Comment (SMillerDev) [inline Formula/gedit.rb:124]: Same

File: Formula/gedit.rb (hunk 2 above)
Comment (tschoonj) [reply Formula/gedit.rb:99]: https://github.com/Homebrew/homebrew-core/pull/60964#discussion_r487119340

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**